.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx --no-install commitlint --edit $1
```

Comment:
```suggestion
yarn lint-staged --edit $1
```

Ditto as above, plus you had commitlint instead of lint-staged. I presume you want lint-staged here.